### PR TITLE
Implement multi insert test case for pkg agent handler - part 1

### PR DESCRIPTION
**Pull request by kevindiu**
### Description:

This PR implements pkg agent ngt multi insert test case (part 1).
I will create another PR to implement the remaining test cases.

### Environment:

- Go Version: 1.18.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pkg/agent/core/ngt/handler/grpc/handler_test.go [hunk 1]
```diff
@@ -3653,9 +3654,113 @@ func Test_server_MultiInsert(t *testing.T) {
...
+	const (
+		name      = "vald-agent-ngt-1" // agent name
+		id        = "uuid-1"           // insert request id
+		intVecDim = 3                  // int vector dimension
+		f32VecDim = 3                  // float32 vector dimension
+	)
+	var (
+		ip     = net.LoadLocalIP()        // agent ip address
+		intVec = []float32{1, 2, 3}       // int vector of the insert request
+		f32Vec = []float32{1.5, 2.3, 3.6} // float32 vector of the insert request
```
Comment (vankichi) [inline pkg/agent/core/ngt/handler/grpc/handler_test.go:3671]: could you please use generate vector function?

Comment (kevindiu) [reply pkg/agent/core/ngt/handler/grpc/handler_test.go]: fixed by removing this variable and use `genF32Req` in test case.

**Review by vankichi — APPROVED** [review]

File: pkg/agent/core/ngt/handler/grpc/handler_test.go [hunk 2]
```diff
@@ -3653,9 +3654,112 @@ func Test_server_MultiInsert(t *testing.T) {
 		beforeFunc func(args)
 		afterFunc  func(args)
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```
Comment (hlts2) [inline pkg/agent/core/ngt/handler/grpc/handler_test.go:3659]: Should we implement this code in the `t.Run` (line. 4311~ )?

Comment (kevindiu) [reply pkg/agent/core/ngt/handler/grpc/handler_test.go:3659]: It is used in `args` field in use case and the context is shared across all of the requests in test case, I think it is ok to implement it here, what do you think?

Comment (hlts2) [reply pkg/agent/core/ngt/handler/grpc/handler_test.go:3659]: Thank you for your comment. I understand.

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**